ui/desktop/app/routes/scopes/scope/projects/settings/index.js

```diff
@@ -4,5 +4,39 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class ScopesScopeProjectsSettingsIndexRoute extends Route {}
+export default class ScopesScopeProjectsSettingsIndexRoute extends Route {
+  @service ipc;
+
+  async model() {
+    let cacheDaemonStatus,
+      formattedCacheVersion,
+      formattedCliVersion,
+      formattedDesktopVersion;
+    const { versionNumber: cliVersion } =
+      await this.ipc.invoke('getCliVersion');
+    formattedCliVersion = `v${cliVersion}`;
+
+    const { desktopVersion } = await this.ipc.invoke('getDesktopVersion');
+    formattedDesktopVersion = `v${desktopVersion}`;
+    try {
+      cacheDaemonStatus = await this.ipc.invoke('cacheDaemonStatus');
```

Comment:
We can still get errors as part of a successful response like this but we can handle this in another ticket if you'd like.

```{
  "status_code": 200,
  "item": {
    "uptime": 2100252029000,
    "socket_address": "/Users/zhiheli/.boundary/socket/daemon.sock",
    "log_location": "/Users/zhiheli/.boundary/cache.log",
    "version": "Boundary v0.17.0 (126c4f55a006525ab4574742c517e7b3b76e0336)",
    "users": [
      {
        "id": "u_1234567890",
        "boundary_instance": {
          "address": "http://localhost:9200",
          "cache_support": "Supported",
          "last_support_check": 20486662000
        },
        "auth_tokens": [
          {
            "id": "at_zFm4apTRvb"
          },
          {
            "id": "at_9DZX9QLWrq",
            "keyringless_references": 1
          }
        ],
        "resources": [
          {
            "name": "resolvable-alias",
            "count": 3,
            "last_error": {
              "error": "cache.(Repository).refreshResolvableAliases: for token \"at_9DZX9QLWrq\": cache.defaultResolvableAliasFunc: unknown, unknown: error #0: ",
              "last_returned": 21007917000
            },
            "refresh_token": {
              "age": 328362859000,
              "last_used": 295585859000
            }
          },
          {
            "name": "target",
            "count": 2,
            "refresh_token": {
              "age": 22450018000,
              "last_used": 20487018000
            }
          },
          {
            "name": "session",
            "count": 0
          }
        ]
      }
    ]
  }
}
```

Reply:
yeah I'll make it a seprate ticket

Reply:
Yeah it is, it's still a successful response and isn't blocking, errors can happen on any individual resource